src/avram/schema_enforcer.cr

```diff
@@ -10,7 +10,7 @@ module Avram::SchemaEnforcer
 
   macro setup(table_name, columns, type, *args, **named_args)
     def self.ensure_correct_column_mappings!
-      return if MODELS_TO_SLIP.includes?(self)
+      return if MODELS_TO_SKIP.includes?(self)
```

Comment:
😂

Reply:
![ship it not](https://user-images.githubusercontent.com/22394/76150687-3387c080-607b-11ea-8c13-8d41739e760d.gif)